gtfs-realtime/spec/en/reference.md

```diff
@@ -187,7 +187,7 @@ The relation between this StopTime and the static schedule.
 | _**Value**_ | _**Comment**_ |
 |-------------|---------------|
 | **SCHEDULED** | The vehicle is proceeding in accordance with its static schedule of stops, although not necessarily according to the times of the schedule. This is the **default** behavior. At least one of arrival and departure must be provided. |
-| **SKIPPED** | The stop is skipped, i.e., the vehicle will not stop at this stop. Arrival and departure are optional. |
+| **SKIPPED** | The stop is skipped, i.e., the vehicle will not stop at this stop. Arrival and departure are optional. SKIPPED value is not propagated to later stops in the same trip, i.e., the vehicle will still stop at subsequent stops unless they also have `schedule_relationship: SKIPPED` in their own `stop_time_update`. In addition, the `trip_update.delay` value is continued to be propagated until it is specified in a subsequent stop. |
```

Comment:
IMHO we should treat `SKIPPED` the same as `NO_DATA` when it comes to propagation - in other words, propagation stops at `SKIPPED` and an agency must provide a stop_time_update with a `delay` or `time` after the `SKIPPED` stop if they want propagation to continue.

Reply:
I revised this with 655687d